org.OpenT2T.Sample.SuperPopular.Thermostat/org.OpenT2T.Sample.SuperPopular.Thermostat.xml

```diff
@@ -10,10 +10,20 @@
     <method name="turnOff"/>
     
     <!-- Get the current ambient temperature -->
-    <method name="getCurrentTemperature">
+    <method name="getAmbientTemperature">
```

Comment:
Wink is not using using thermostat schema. Nest is the first thermostat being added here. 
Wink is using org.OpenT2T.Sample.SuperPopular.TemperatureSensor schema. Am I missing anything ?

---
In reply to: [68283197](https://github.com/openT2T/translators/pull/22#discussion_r68283197) [](ancestors = 68283197)